ko/admin/config.rst

```diff
@@ -597,11 +597,11 @@ CUBRID 설치 시 생성되는 기본 데이터베이스 환경 설정 파일(**
     *   **$CUBRID/conf/hosts.conf** 파일의 형식은 **/etc/hosts** 와 동일하나 다음과 같은 몇 가지 제약이 있다.
 
         * **IPv4** 주소 만을 허용한다 (**IPv6** 는 허용하지 않는다).
-        * 아래와 같은 형식의 **alias** 는 허용하지 않는다. ::
+        * **alias** 는 허용하지 않는다. ::
 
            172.31.0.1 host1 alias1 alias2
```

Comment:
Thank you for the review.

I think users could need this. 

Because it is not allowed only in the user host of CUBRID, so detailed example would be necessary not to use 'alias' function. I think this example could help users not to make mistakes.

Reply:
ok